packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { cloudinaryPlusBackground } from '@dailydotdev/shared/src/lib/image';
+import { MainFeedPageProps } from '../MainFeedPage';
+import { PlusHeader } from './PlusHeader';
+
+export default function PlusLayout({
+  children,
+}: MainFeedPageProps): ReactElement {
+  return (
+    <main className="relative flex h-screen flex-col bg-blur-glass">
+      <img
+        src={cloudinaryPlusBackground}
+        alt="Plus background"
+        className="absolute inset-0 -z-1 h-full w-full"
+        aria-hidden
+      />
```

Comment:
Should we create a global component for these sort of images?
Non-blocking

Reply:
Let's add it to backlog :)

Reply:
👍